quotes_gen/main.go

```
@@ -0,0 +1,88 @@
+
+	mux := http.NewServeMux()
+
+	mux.HandleFunc("GET /v1/quotes", func(w http.ResponseWriter, r *http.Request) {
+		res := quotesRes{Quote: quoteGen.getQuote()}
+		resJson, err := json.Marshal(res)
+		if err != nil {
+			logger.Error().Err(err).Send()
+			http.Error(w, err.Error(), http.StatusInternalServerError)
+			return
+		}
+		w.Write(resJson)
```

golangci-lint (1.64.8)

Error return value of `w.Write` is not checked

(errcheck)